### ISSUE TITLE
Tests of other third-party libraries

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal-testsuite-dockerfiles#26]
In my `CMakeCache.txt`, I can read:
```
CGAL_SUPPORTING_3RD_PARTY_LIBRARIES:INTERNAL=GMP;GMPXX;MPFR;ZLIB;OpenGL;LEDA;MPFI;RS;RS3;OpenNL;Eigen3;BLAS;LAPACK;QGLViewer;ESBTL;Coin3D;NTL;IPE;TAUCS
```

I wonder if `GMPXX` is tested.

The Docker testsuite is supposed to replace the legacy testsuite, and I think that the following should be tested, at least on some platforms:
  - [MPFI](http://mpfi.gforge.inria.fr/), with [RS](http://vegas.loria.fr/rs/)
  - [ESBTL](http://esbtl.sourceforge.net/) (headers only)
  - [NTL](http://www.shoup.net/ntl/)
  - ipe (there are packages for Fedora, at least).

TAUCS should no longer be tested.

**Comment by bo0ts** [comment]
I added ESBTL, NTL and IPE to some platforms. I did not add RS yet, because I have doubts that the license actually allows it. If you think RS is OK to add, I'll add it to some platforms.

**Comment by lrineau** [comment]
Fixed by #44.
